app/views/partials/_category_navbar.html.haml

```diff
@@ -1,4 +1,6 @@
-- categories = Category.all 
-- categories.each do |category|
-    %div
-        = link_to category.name, category_path(category)
\ No newline at end of file
+- categories = Category.all
```

Comment:
This is a view

Reply:
We render this partial at the application.layout, can you help us how can we pass the data down there?